sw/host/spiflash/README.md

```diff
@@ -73,6 +61,15 @@ If there are two FPGAs or multiple valid targets attached at the same time, it i
 
 ```console
 $ cd ${REPO_TOP}
-$ build-bin/sw/host/spiflash/spiflash \
-  --input build-bin/sw/device/examples/hello_world/hello_world_fpga_nexysvideo.bin
+$ build-out/sw/host/spiflash/spiflash --input ${FLASH_BIN}
```

Comment:
We should keep using the `build-bin` versions of the binary, I think.

Reply:
Done.